packages/blade/src/components/Spinner/BaseSpinner/BaseSpinner.tsx

```diff
@@ -51,10 +51,10 @@ const getColor = ({ color, theme }: { color: BaseSpinnerProps['color']; theme: T
   if (color && color === 'white') {
     return getIn(theme.colors, 'interactive.icon.staticWhite.subtle');
   }
-  if (color && color !== 'primary') {
-    return getIn(theme.colors, `feedback.background.${color}.intense`);
+  if (color && color !== 'neutral') {
+    return getIn(theme.colors, `interactive.icon.${color}.subtle`);
   }
-  return getIn(theme.colors, 'surface.background.primary.intense');
+  return getIn(theme.colors, 'interactive.icon.gray.subtle');
 };
```

Comment:
Spinner should have surface.icon* colors no? since its not interactive element 🤔

Reply:
Good point... on figma its using interactive though

Reply:
Asking design